public/app/features/dashboard/containers/DashboardPage.tsx

```diff
@@ -74,6 +77,7 @@ export class DashboardPage extends PureComponent<Props, State> {
     scrollTop: 0,
     updateScrollTop: null,
     rememberScrollTop: 0,
+    inspectPanel: null,
```

Comment:
I'd probably move this to the DashboardPanel, closer to where it actually belongs

Reply:
I think a single instance on the page makes more sense then a flag on each panel -- we should only have one for the whole dashboard/page